src/index.tsx

```diff
@@ -20,7 +25,10 @@ const persister = persistStore(store);
 const provider = (
   <ReduxProvider store={store}>
     <PersistGate persistor={persister} loading={<div>Loading NeoDash...</div>}>
-      <Application />
+      <Router>
```

Comment:
We can extract lines 28-31 and move them into `src/extensions/hive` somewhere. Then we can dynamically plug in the router here if the Hive extension is enabled.